### PR TITLE
Remove expectations

### LINKED ISSUE #2967: Reference to deleted BuildStepStatus
This ticket is a migrated Trac ticket [3401](http://trac.buildbot.net/ticket/3401)

People contributed to the original ticket: @unknown_contributor, @rutsky, @unknown_contributor
Ticket created on: `Jan 05 2016`
Ticket last modified on: `Feb 23 2016`

---

The following is referencing the recently deleted I[[BuildStepStatus]] interface in 0.9.0b5, causing a runtime exception:

buildbot/master/buildbot/status/master.py:182

---

_Comment from_: @unknown_contributor
_Date_: `Feb 22 2016`

I'm seeing this exception in the master log with 0.9.0b7 as well.

    2016-02-22 14:14:07-0800 [-] Unhandled Error
    	Traceback (most recent call last):
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\twisted\application\app.py", line 392, in startReactor
    	    self.config, oldstdout, oldstderr, self.profiler, reactor)
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\twisted\application\app.py", line 313, in run[[ReactorWithLogging]]
    	    reactor.run()
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\twisted\internet\base.py", line 1192, in run
    	    self.mainLoop()
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\twisted\internet\base.py", line 1201, in mainLoop
    	    self.run[[UntilCurrent]]()
    	--- <exception caught here> ---
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\twisted\internet\base.py", line 824, in run[[UntilCurrent]]
    	    call.func(*call.args, **call.kw)
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\buildbot\buildslave\base.py", line 317, in _missing_timer_fired
    	    text += "%s\n" % status.getURL[[ForThing]](self.slave_status)
    	  File "C:\[[TrionBuildbot]]\python\lib\site-packages\buildbot\status\master.py", line 182, in getURL[[ForThing]]
    	    if interfaces.I[[BuildStepStatus]].providedBy(thing):
    	exceptions.[[AttributeError]]: 'module' object has no attribute 'I[[BuildStepStatus]]'

	

---
_Comment from_: @rutsky
_Date_: `Feb 22 2016`

@gtmacdonald, can you provide configuration example in which this issue can be reproduced?

---
_Comment from_: @unknown_contributor
_Date_: `Feb 23 2016`

It's very large. I'll see if I can narrow it down.

### PR DESCRIPTION
**Pull request by ewongbb**
fixes #2967.

Note: while changing the IBuildStepStatus mentions.. I noticed stepETAUpdates wasn't being used
anywhere so I removed it.

### CODE REVIEW

**Comment by tardyp** [comment]
please fix the spelling

File: master/buildbot/interfaces.py [hunk 1]
```diff
@@ -803,7 +803,7 @@ def getLogs():
         """Return a list of logs that describe the build as a whole. Some
         steps will contribute their logs, while others are are less important
         and will only be accessible through dictionary obtained from
-        py:meth:`getSteps`.  Each log is an object which implements the IStatusLog
+        get steps method.  Each log is an object which implements the IStatusLog
```
Comment (tardyp) [inline master/buildbot/interfaces.py]: no you need to put getSteps in https://github.com/buildbot/buildbot/blob/master/common/code_spelling_ignore_words.txt

File: master/docs/spelling_wordlist.txt [hunk 2]
```diff
@@ -310,6 +310,7 @@ getChoices
 getfqdn
 getProperties
 getRecipients
+getSteps
```
Comment (tardyp) [inline master/docs/spelling_wordlist.txt]: wrong file. please remove this commit.

**Comment by tardyp** [comment]
please squash the messed up history..

**Comment by ewongbb** [comment]
Closing this PR in favor of https://github.com/buildbot/buildbot/pull/3620

**Pull request closed without merging**